content/nginx/deployment-guides/single-sign-on/active-directory-federation-services.md

```diff
@@ -1,19 +1,17 @@
 ---
-description: Enable OpenID Connect-based single sign-on (SSO) for applications proxied by NGINX Plus, using Microsoft AD FS as the identity provider (IdP).
-type:
-- how-to
 title: Single Sign-On with Microsoft Active Directory FS
+description: Enable OpenID Connect-based single sign-on (SSO) for applications proxied by NGINX Plus, using Microsoft AD FS as the identity provider (IdP).
```

Comment:
Thought we were not using this field any more

Reply:
No, the description field is how our pages generate meta descriptions which are important for SEO.

Nobody's prioritized writing them, but that doesn't mean they're not important.